JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -2892,10 +2828,33 @@
     plot_data <- list()
     
     for (cf in 1:coefs) {
+      
+      # this is a mess but the stanova::stanova_samples returns an incomplete variable names
+      coefs_trend <- attr(samples, "estimate")
+      coefs_trend <- gsub("trend ('", "", coefs_trend, fixed = TRUE)
+      coefs_trend <- gsub("')", "", coefs_trend, fixed = TRUE)
+      coefs_trend <- strsplit(coefs_trend, ",")
+      
+      coefs_name <-
+        paste(.unv(unlist(
+          strsplit(dimnames(samples)$Parameter[cf], ",")
+        )), collapse = ":")
+      coefs_name <- gsub(" ", "", coefs_name, fixed = TRUE)
+      coefs_name <-
+        .mmVariableNames(coefs_name, options$fixedVariables)
+      
+      if(length(coefs_trend) > 0){
+        for(cft in coefs_trend){
```

Comment:
```suggestion
        for(cft in coefs_trend){
```
the loop is not entered if `length(coefs_trend)` is 0.

Reply:
yes, that was the intention. Is there a problem with that?

Reply:
the loop `for(cft in coefs_trend){` is not entered if the length of `coefs_trend` is 0, so the if statement `if(length(coefs_trend) > 0){` is redundant. Compare:
```r
for (a in list(a = 1))
  print(a)

for (a in list())
  print(a)
```

Reply:
oh, I see now. Thanks